announcements/1.14.0.md

```diff
@@ -0,0 +1,87 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.14.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
```

Comment:
1. Implement syscall interception via LXC seccomp notification support. This enables Anbox to handle different syscalls like setpriority or sched_set_scheduler for processes inside the Android container.
2. Unify the pointer input model by using pointer events to handle mouse and touch events.
3. Switch the memory allocator from the default Glibc based to the tcmalloc to decrease the memory footprint and optimize the memory consumption for Anbox session.

Reply:
Added under "Other" and to the release notes.